README.md

```diff
@@ -1,5 +1,3 @@
-[![Build Status](https://travis-ci.org/google/google-api-dotnet-client.svg?branch=master)](https://travis-ci.org/google/google-api-dotnet-client)
```

Comment:
I think that you removed it by mistake...  maybe as a result of bad merge :-1:  :)